x/swap/types/msgs.go

```diff
@@ -86,3 +71,74 @@ func (msg MsgAddLiquidity) GetSigners() []sdk.AccAddress {
 func (msg MsgAddLiquidity) GetSwapTokenPair() string {
 	return msg.MaxBaseTokens.Denom + "_" + msg.QuoteTokens.Denom
 }
+
+// MsgTokenOKTSwap define the message for swap between token and DefaultBondDenom
+type MsgTokenOKTSwap struct {
+	SoldTokenAmount      sdk.DecCoin    `json:"sold_token_amount"`       //Amount of Tokens sold.
+	MinBoughtTokenAmount sdk.DecCoin    `json:"min_bought_token_amount"` //Minimum token purchased.
+	Deadline             int64          `json:"deadline"`                //Time after which this transaction can no longer be executed.
+	Recipient            sdk.AccAddress `json:"recipient"`               //if give Recipient address,transfers Tokens to recipient.default recipient is sender
+	Sender               sdk.AccAddress `json:"sender"`                  //sender
+}
+
+// NewMsgTokenOKTSwap is a constructor function for MsgTokenOKTSwap
+func NewMsgTokenOKTSwap(
```

Comment:
Function name appears OKT is not flexible enough

Reply:
just for token_okt pair, TokenToTokenSwap for token_token